inc/options/cdn.php

```diff
@@ -1,157 +1,348 @@
 <?php
 namespace W3TC;
 
-if ( !defined( 'W3TC' ) )
+if ( ! defined( 'W3TC' ) ) {
 	die();
+}
 
 // when separate config is used - each blog has own uploads
-// so nothing to upload from network admin
-$upload_blogfiles_enabled = $cdn_mirror || !is_network_admin() ||
-	!Util_Environment::is_using_master_config();
+// so nothing to upload from network admin.
+$upload_blogfiles_enabled = $cdn_mirror || ! is_network_admin() || ! Util_Environment::is_using_master_config();
 
 ?>
-<?php include W3TC_INC_DIR . '/options/common/header.php'; ?>
+<?php require W3TC_INC_DIR . '/options/common/header.php'; ?>
 <p id="w3tc-options-menu">
-	<?php _e( 'Jump to:', 'w3-total-cache' ); ?>
-	<a href="#toplevel_page_w3tc_general"><?php _e( 'Main Menu', 'w3-total-cache' ); ?></a> |
-	<a href="#general"><?php _e( 'General', 'w3-total-cache' ); ?></a> |
-	<a href="#configuration"><?php _e( 'Configuration', 'w3-total-cache' ); ?></a> |
-	<a href="#advanced"><?php _e( 'Advanced', 'w3-total-cache' ); ?></a> |
-	<a href="#notes"><?php _e( 'Note(s)', 'w3-total-cache' ); ?></a>
+	<?php esc_html_e( 'Jump to:', 'w3-total-cache' ); ?>
+	<a href="#toplevel_page_w3tc_general"><?php esc_html_e( 'Main Menu', 'w3-total-cache' ); ?></a> |
+	<a href="#general"><?php esc_html_e( 'General', 'w3-total-cache' ); ?></a> |
+	<a href="#configuration"><?php esc_html_e( 'Configuration', 'w3-total-cache' ); ?></a> |
+	<a href="#advanced"><?php esc_html_e( 'Advanced', 'w3-total-cache' ); ?></a> |
+	<a href="#notes"><?php esc_html_e( 'Note(s)', 'w3-total-cache' ); ?></a>
 </p>
 
 <p>
-	<?php echo sprintf(
-	__( 'Content Delivery Network support via %1$s is currently %2$s.', 'w3-total-cache' ),
-	'<strong>'.Cache::engine_name( $this->_config->get_string( 'cdn.engine' ) ).'</strong>',
-	'<span class="w3tc-' . ( $cdn_enabled ? 'enabled">' . __( 'enabled', 'w3-total-cache' ) : 'disabled">' . __( 'disabled', 'w3-total-cache' ) ) . '</span>'
-); ?>
+	<?php
+	echo wp_kses(
+		sprintf(
+			// translators: 1 HTML strong tag containing CDN Engine value, 2 HTML span tag containing CDN Engine enabled/disabled value.
+			__(
+				'Content Delivery Network support via %1$s is currently %2$s.',
+				'w3-total-cache'
+			),
+			'<strong>' . Cache::engine_name( $this->_config->get_string( 'cdn.engine' ) ) . '</strong>',
+			'<span class="w3tc-' . ( $cdn_enabled ? 'enabled">' . __( 'enabled', 'w3-total-cache' ) : 'disabled">' . __( 'disabled', 'w3-total-cache' ) ) . '</span>'
```

Comment:
Use `esc_html__()`.

Reply:
Changed to esc_html__ for both instances